confdb/confdb.go

```diff
@@ -460,20 +462,20 @@ func getPlaceholders(viewStr string) map[string]bool {
 	return placeholders
 }
 
-// View returns a view from the confdb.
-func (db *Confdb) View(view string) *View {
-	return db.views[view]
+// View returns a view from the confdb schema.
+func (s *Schema) View(view string) *View {
+	return s.views[view]
 }
 
-// View carries access rules for a particular view in a confdb.
+// View carries access rules for a particular view in a confdb schema.
 type View struct {
 	Name   string
 	rules  []*viewRule
-	confdb *Confdb
+	schema *Schema
 }
 
-func (v *View) Confdb() *Confdb {
-	return v.confdb
+func (v *View) ConfdbSchema() *Schema {
```

Comment:
just Schema now ?

Reply:
maybe it's worth leaving it to disambiguate from the storage schema?

Reply:
I think the type helps with which is which